frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -33,21 +32,21 @@ const EventIndicator = styled.div<{ selected?: boolean; eventPresent?: boolean }
   background-color: ${(props) => (props.selected ? 'white' : '#468751')};
 `
 
-export const ClickableDateContainer = (props: { date: Date; eventPresent?: boolean; facilityId: number }) => {
+export const ClickableDateContainer = (props: {
+  date: Date
+  eventPresent?: boolean
+  facilityId: number
+  overlay?: boolean
+  onClickDate: (date: Date) => void
+}) => {
   const dispatch = useDispatch()
   const history = useHistory()
   const { clickedDate, processedDates } = useSelector((state: RootState) => state.calendar)
   const assignedDateMonth = new Date(props.date.getFullYear(), props.date.getMonth(), props.date.getDate())
```

Comment:
is this necessary? since props.date is already of date format? does this deconstruct the date and create a new one?